src/pages/VerifySettingsPage/VerifySettingsPage.tsx

```diff
@@ -0,0 +1,54 @@
+import { useMemo, useState, useCallback } from "react";
+import get from "lodash/get";
+import {
+  useDeskproAppClient,
+  useDeskproAppEvents,
+} from "@deskpro/app-sdk";
+import { getCompanyInfoService } from "../../services/peoplehr";
+import { VerifySettings } from "../../components";
+import type { FC } from "react";
+import type { Maybe, Settings } from "../../types";
+import type { Company } from "../../services/peoplehr/types";
+
+const VerifySettingsPage: FC = () => {
+  const { client } = useDeskproAppClient();
+  const [companyInfo, setCompanyInfo] = useState<Maybe<Company>>(null);
+  const [settings, setSettings] = useState<Maybe<Settings>>(null);
+  const [isLoading, setIsLoading] = useState<boolean>(false);
+  const [error, setError] = useState<string>("");
+  const API_KEY = useMemo(() => get(settings, ["api_key"]), [settings]);
+  const errorMessage = useMemo(() => "Failed to connect to SimpleMDM, settings seem to be invalid", []);
```

Comment:
theres no need for an useMemo for static values, it just adds more memory to the heap

Reply:
done